client/app/app.translate.js

```diff
@@ -0,0 +1,13 @@
+import localeEN from "./locales/locale-en.json";
+import localeDE from "./locales/locale-de.json";
+
+let AppTranslate = ($translateProvider) => {
+  "ngInject";
+  $translateProvider
+  .translations("en", localeEN)
```

Comment:
...or underscores (e.g. `en_US`) to make http://userguide.icu-project.org/locale happy